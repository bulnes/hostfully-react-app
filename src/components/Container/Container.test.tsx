import { render } from "@testing-library/react";
import { Container } from "./Container.component";

describe("Component: Container", () => {
  it("should be defined", () => {
    expect(Container).toBeDefined();
  });

  it("should render correctly", () => {
    const wrapper = render(<Container>Container</Container>);
    expect(wrapper).toBeDefined();
  });

  it("should render children correctly", () => {
    const wrapper = render(<Container>Container</Container>);
    expect(wrapper.getByText("Container")).toBeDefined();
  });
});
