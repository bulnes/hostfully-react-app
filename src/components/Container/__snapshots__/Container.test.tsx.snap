// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: Container should be represented by the snapshot 1`] = `
<div>
  <div
    class="container mx-auto"
  >
    Container
  </div>
</div>
`;
